### PR TITLE
Document workflow runs

### PR DESCRIPTION
**Pull request by rubhanazeem**

### CODE REVIEW

File: xml/obs_scm_ci_workflow_integration.xml [hunk 1]
```diff
@@ -778,6 +778,25 @@ workflow:
       </figure>
     </sect2>
 
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.workflow_runs">
+      <title>Workflow Runs</title>
+
+      <para>For every SCM event, OBS record the request, and the response sent back to the SCM.
+      These records are stored in OBS in the form of Workflow runs. OBS will show details of every
+      event received from SCM. If an error occures in any of the workflow steps, the
+      workflow run will record the same response as it was sent to the SCM.</para>
```
Comment (hennevogel) [inline xml/obs_scm_ci_workflow_integration.xml:787]: Sounds like 2 times the same sentence. Maybe apart from *what* you should explain *why* we are doing this and *how* this is useful to the people reading this.

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: xml/obs_scm_ci_workflow_integration.xml (hunk 1 above)
Comment (rubhanazeem) [reply xml/obs_scm_ci_workflow_integration.xml]: I've pushed some updates. Please let me know if you have more suggestions. :)

File: xml/obs_scm_ci_workflow_integration.xml [hunk 2]
```diff
@@ -778,6 +778,25 @@ workflow:
       </figure>
     </sect2>
 
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.workflow_runs">
+      <title>Workflow Runs</title>
+
+      <para>For every SCM event, OBS records the request and the response sent back to the SCM.
+      These records are stored in OBS in the form of Workflow runs. The main reason for these
```
Comment (dmarcoux) [inline xml/obs_scm_ci_workflow_integration.xml:785]:
```suggestion
      These records are stored in OBS in the form of workflow runs.
```

File: xml/obs_scm_ci_workflow_integration.xml [hunk 3]
```diff
@@ -778,6 +778,25 @@ workflow:
...
+      workflow runs is to help the users to debug their workflows. If an error occurs in any of
```
Comment (dmarcoux) [inline xml/obs_scm_ci_workflow_integration.xml:786]:
```suggestion
      This helps with debugging workflows. If an error occurs in any of
```

**Review by dmarcoux — CHANGES_REQUESTED** [review]

**Review by dmarcoux — APPROVED** [review]

**Review by eduardoj — APPROVED** [review]

**Pull request merged**